Dockerfile

```diff
@@ -6,11 +6,14 @@
 FROM golang:1.9-alpine AS dist
 
 
-RUN apk add --update curl
+RUN apk add --update curl git
+
+# Download and build rclone.
+RUN go get -u -v github.com/ncw/rclone
```

Comment:
@alnikyur that's not how we install stuff. You can use something like this for debugging or temp stuff, but not for production grade Docker images.

The problems are:
- `go get` uses `master` which can easily be unstable. You should use latest stable release ([this](https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c) may be helpful).
- [rclone license](https://github.com/ncw/rclone/blob/v1.41/COPYING) is violated, because you've just installed binary. You should copy license inside image as we do for restic.

Reply:
So, i need to install `rclone` from source code like `restic`?

Reply:
@alnikyur with `go get` you're installing rclone from source code too (you may read how `go get` works). The matter is in the sources version you're compiling.